src/com/ean/clarity/Main.java

```diff
@@ -0,0 +1,53 @@
+package com.ean.clarity;
+
+import com.ean.clarity.util.Baugus;
+
+import java.util.Scanner;
+
+/**
+ * Does main stuff.
+ */
+public class Main {
+
+    private static final String CLARITY_URL = "http://clarity.expedia.biz";
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Timesheet timesheet = new Timesheet();
+
+        // TODO: Change these submits to trigger on Tab key instead of Enter key (for MacBook Pro Tab8 support)
+
+        log("Monday hours: ");
+        timesheet.setMondayHours(scanner.nextInt());
+        log("Tuesday hours: ");
+        timesheet.setTuesdayHours(scanner.nextInt());
+        log("Wednesday hours: ");
+        timesheet.setWednesdayHours(scanner.nextInt());
+        log("Thursday hours: ");
+        timesheet.setThursdayHours(scanner.nextInt());
+        log("Friday hours: ");
+        timesheet.setFridayHours(scanner.nextInt());
+
+        if (!timesheet.isValid()) {
+            log("Invalid timesheet. Please do not enter any number other than 8.");
```

Comment:
To prevent consistency issues in the future should we ever switch to 9-hour standard workdays, this should reference VALID_HOURS from TimeSheet.java